palvelutarjotin/urls.py

```diff
@@ -35,32 +33,22 @@
 #
 # Kubernetes liveness & readiness probes
 #
-@require_http_methods(["GET", "HEAD"])
-def healthz(*args, **kwargs):
-    return HttpResponse(status=200)
-
-
-@require_http_methods(["GET", "HEAD"])
```

Comment:
`@require_GET` or `@require_http_methods(["GET", "HEAD"])` or `@require_safe` could be used here to get rid of SonarCloud security hotspot "Make sure allowing safe and unsafe HTTP methods is safe here." / https://rules.sonarsource.com/python/RSPEC-3752/